From 355f56a0f1889a0d2fe59588c011b1d743e16329 Mon Sep 17 00:00:00 2001
From: Larisa Grigore <larisa.grigore@nxp.com>
Date: Mon, 21 Dec 2020 11:56:47 +0200
Subject: [PATCH 20/76] s32g274rdb: Fix compile issue
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

board/freescale/s32-gen1/common.c:159:2:
error: too many arguments to function ‘puts’
  159 |  puts("Board:\tNXP %s-RDB\n", get_s32g2_deriv_name());

Issue: ALB-6229
Signed-off-by: Larisa Grigore <larisa.grigore@nxp.com>
---
 board/freescale/s32-gen1/common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/freescale/s32-gen1/common.c b/board/freescale/s32-gen1/common.c
index e43189ec10..52659232e1 100644
--- a/board/freescale/s32-gen1/common.c
+++ b/board/freescale/s32-gen1/common.c
@@ -156,7 +156,7 @@ int checkboard(void)
 #if defined(CONFIG_TARGET_S32G274AEVB)
 	printf("Board:\tNXP %s-EVB\n", get_s32g2_deriv_name());
 #elif defined(CONFIG_TARGET_S32G274ARDB)
-	puts("Board:\tNXP %s-RDB\n", get_s32g2_deriv_name());
+	printf("Board:\tNXP %s-RDB\n", get_s32g2_deriv_name());
 #elif defined(CONFIG_TARGET_S32G274ABLUEBOX3)
 	puts("Board:\tNXP S32G274A BlueBox3\n");
 #elif defined(CONFIG_TARGET_S32G274ASIM)
-- 
2.17.1

